### PR TITLE
Student interface

### PR DESCRIPTION
**Pull request by prathamesh920**
Student test interface and login page improved.
Made time-left component fixed at the top
Reset button for code question shifted inside the editor panel.
Reduced the empty space at the top and few improvement in look and feel.

### CODE REVIEW

**Comment by prathamesh920** [comment]
mcq
code
code_bottom
complete

**Comment by prabhuramachandran** [comment]
1. Can you fix the conflicts and spelling I suggest.
2. Remove the "Python code question" thing below the question title,
3. Put the question type in brief after the title (MCQ), (MCC) etc.
4. Submit below the code and not below output.
5. The yaksh online test title is overlapping the question, make it a small header.

File: yaksh/templates/yaksh/question.html [hunk 1]
```diff
@@ -242,7 +259,7 @@ <h4 class="modal-title" id="myModalLabel">File not selected</h4>
   editor.setValue('{{ last_attempt|escape_quotes|safe }}')
 
   function reset_editor() {
-      editor.setValue('{{ last_attempt|escape_quotes|safe }}');
+      editor.setValue('');
```
Comment (prathamesh920) [inline yaksh/templates/yaksh/question.html]: For now reverted back to the original code.
The problem is if I write a code and submit and then want to reset, then the reset will show the same code.

**Comment by prathamesh920** [comment]
code_modified

**Comment by prathamesh920** [comment]
Resolved all issues

**Comment by ankitjavalkar** [comment]
The interface looks very nice!

**Pull request merged**